README.md

```diff
@@ -29,30 +28,54 @@ $ pip install graphix[extra]
 ```
 this will install `graphix` and inteface for [IBMQ](https://github.com/TeamGraphix/graphix-ibmq) and [Perceval](https://github.com/TeamGraphix/graphix-perceval) to run MBQC patterns on superconducting and optical QPUs and their simulators.
 
-<!-- We are currently adding more quantum device interface.
-Please suggest in [issues](https://github.com/TeamGraphix/graphix/issues) if you have any particular device in mind! -->
 
+## Using graphix
+
+### generating pattern from a circuit
+```python
+from graphix import Circuit
+circuit = Circuit(4)
+circuit.h(0)
+...
+pattern = circuit.transpile()
+pattern.draw_graph()
+```
+<img src="https://github.com/TeamGraphix/graphix/assets/33350509/de17c663-f607-44e2-945b-835f4082a940" alt="logo" width="750">
+
+<small>note: this graph is generated from QAOA circuit, see [our example code](examples/qaoa.py). Arrows indicate the [*information flow*](https://journals.aps.org/pra/abstract/10.1103/PhysRevA.74.052310) of MBQC and dashed lines are the other edges of the graph. the vertical dashed partitions and the labels 'l:n' below indicate the execution *layers* or the order in the graph (measurements should happen from left to right, and nodes in the same layer can be measured simultaneously), based on the partial order associated with the (maximally-delayed) flow. </small>
```

Comment:
Isn't it a "causal flow" rather than an "information flow"? Flow does not always transfer information especially in generalized flow.

Reply:
good point, I thought information flow is more accessible to non-specialist (here it's only referring to flow) but let's be precise. updated this part.

Reply:
Thanks for the update!

>  the vertical dashed partitions and the labels 'l:n' below indicate the execution layers or the order in the graph (measurements should happen from left to right, and nodes in the same layer can be measured simultaneously), based on the partial order associated with the (maximally-delayed) flow.

This part explains well what flow is in MBQC